packages/legend-query-builder/src/stores/QueryBuilderValueSpecificationBuilderHelper.ts

```diff
@@ -148,7 +148,9 @@ export const buildPropertyExpressionChain = (
     graph.getTypicalMultiplicity(TYPICAL_MULTIPLICITY_TYPE.ONE),
   );
   newPropertyExpression.func = propertyExpression.func;
-  newPropertyExpression.parametersValues = propertyExpression.parametersValues;
+  newPropertyExpression.parametersValues = [
```

Comment:
yes, this is naturally included in all the lambda roundtrip tests we have. You should see a warning from `mobx` if you don't do this since you're effectively directly modifying an observable array `propertyExpression.parametersValues`